plugins/synaptics-cape/fu-synaptics-cape-device.c

```diff
@@ -418,6 +530,8 @@ fu_synaptics_cape_device_setup(FuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	g_debug("InReportViaInterrupt: %s", self->InReportViaInterrupt ? "yes" : "no");
```

Comment:
not this.

Reply:
okay. 
BTW, can tool dump all flag value and friendly name in log?

Reply:
Yes, if you register a private flag then the fu_device_to_string() knows how to print it.